ps5ControllerApp/ps5ControllerApp/ViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ViewController.swift
+//  ps5ControllerApp
+//
+//  Created by Rıdvan Mertoğlu on 2.05.2021.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    @IBOutlet weak var collectionView: UICollectionView!
+    @IBOutlet weak var titleLabel: UILabel!
+    
+    @IBOutlet private weak var controllerTabView: UIView!
+    @IBOutlet private weak var devicesTabView: UIView!
+    @IBOutlet private weak var mousesTabView: UIView!
+    
+    let products: [Product] = [.init(image:UIImage(named: "blackPsController")),.init(image: UIImage(named: "whitePsController"))]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let backgroundImage = UIImageView(frame: UIScreen.main.bounds)
+        backgroundImage.image = UIImage(named: "bg.png")
+        backgroundImage.contentMode = UIView.ContentMode.scaleAspectFill
+        self.view.insertSubview(backgroundImage, at: 0)
+        collectionView.dataSource = self
+        collectionView.delegate = self
+        
+        collectionView.register(.init(UINib(nibName: "ControllerCollectionViewCell", bundle: .main)), forCellWithReuseIdentifier: "psControllerIdentifier")
```

Comment:
bunu her cell için yapıyosun, extension'a taşıyıp sadece collectionView.register(type: ControllerCollectionViewCell.self) şeklinde nası yaparsın? Hep her cell'in nibName'i class adı, identifier'ı da class adı olsun